build.ps1

```diff
@@ -6,6 +6,7 @@ param(
     [switch]$MacOs,
     [switch]$Osx,
     [switch]$Rpi,
+    [switch]$Arm64,
```

Comment:
@gregolsky when dealing with this rename this to LinuxArm64, in .NET 5 we will have WindowsArm64 as well.

Reply:
ok gotcha